app-creator/client/src/redux/reducer.ts

```diff
@@ -212,3 +219,30 @@ function updateUI(widget: HTMLElement, template: { [key: string]: any }) {
     widget.setAttribute(attr, template[widget.id].uniqueAttributes[attr]);
   }
 }
+
+function getBackgroundColor(style: { [key: string]: any }) {
+  // Stringified number from 0 - 100 (only integers) or an empty string.
+  let backgroundOpacity = style.backgroundOpacity;
+
+  // Empty string case.
+  if (backgroundOpacity === '') {
+    backgroundOpacity = '0';
+  }
+
+  const number = parseInt(backgroundOpacity);
```

Comment:
Can we rename `backgroundOpacity` to `backgroundOpacityStr` and `number` to `backgroundOpacity`, so it's more explicit?